pkg/processor/types.go

```diff
@@ -26,3 +26,15 @@ type Configuration struct {
 	functionconfig.Config
 	PlatformConfig *platformconfig.Config
 }
+
+type OffsetData struct {
+	Topic       string `json:"topic"`
+	Partition   int32  `json:"partition"`
+	Offset      int64  `json:"offset"`
+	TriggerName string `json:"trigger_name"`
+	Err         error
+}
+
+func (o *OffsetData) HasTriggerName() bool {
+	return o.TriggerName == ""
+}
```

Comment:
why is it here? move to kafka types and name it something like `ControlMessageAttributesExplicitAck`

Reply:
Again, removed since irrelevant to this PR.